src/migrateContentServer.test.ts

```diff
@@ -18,6 +19,54 @@ jest.mock(`./4.3_to_5.0/generateId`, () => {
   };
 });
 
+/**
+ * Contains three legacy bookmark widgets.
+ * 1. Widget with an invalid container container key.
+ * 2. Widget with a filter on an invalid hierarchy.
+ * 3. A valid widget.
+ */
+const bookmarkContentWithInvalidWidgets = {
```

Comment:
It would be nice to move this to its own test resource (e.g. `invalidLegacyWidgets.ts`).

Reply:
Done, I split this out to `__test_resources__/invalidLegacyWidgets.ts`